nuxeo-common/src/main/java/org/nuxeo/common/utils/StringUtils.java

```diff
@@ -192,4 +195,21 @@ public static String expandVars(String expression, Map<?, ?> properties) {
         return Vars.expand(expression, properties);
     }
 
+    /**
+     * Returns a json string representation of the object.
+     * <p>
+     * This helper does not check for circular reference and will lead to a stack overflow if any.
+     *
+     * @param object the object
+     * @return a json string or null if object is null
+     * @throws IOException
+     * @since 10.3
+     */
+    public static String toJson(Object object) throws IOException {
```

Comment:
I'm not a fan of trivial methods in nuxeo-common. I'd really like nuxeo-common to disappear if we could, as we're mostly reinventing things that exist elsewhere. Please define this method where you need it but it's not worth augmenting nuxeo-common for it imho.

Reply:
Ok, I had a dedicated helper in the service in the first place but it sounded weird to have a dep on jackson in `nuxeo-runtime`. If we have a dedicated WebUIConfService, I guess it'd be more natural